Dockerfile.prod

```diff
@@ -14,6 +14,7 @@ RUN apt-get update \
                        libffi-dev \
                        libssl-dev \
                        redis-tools \
+                       cron \
```

Comment:
it looks like cron is already in the `metabrainz/python:3.6` image, I don't know if you want to add it here again explicitly to make it clear that we use it

Reply:
Ok, we now understand that this is not needed.